code/game/gamemodes/objective.dm

```diff
@@ -614,7 +614,7 @@ GLOBAL_LIST_EMPTY(possible_items_special)
 		absorbedcount += changeling.absorbedcount
 
 	for(var/datum/antagonist/changeling/changeling2 in GLOB.antagonists)
-		if(!changeling2.owner || !changeling2.stored_profiles || changeling2.absorbedcount < absorbedcount)
+		if(!changeling2.owner || changeling2 == owner || changeling2.owner == owner || !changeling2.stored_profiles || changeling2.absorbedcount < absorbedcount)
```

Comment:
Antag datum will never be a mind, no need for the first compare.

Reply:
Some objectives also get added to antag datums, and I noticed the check_completion was running twice so I added this in case the objective's owner was the datum, but i'll test it without that check

Reply:
Antagonists have separate objectives list yes, but the owner will never actually be a mind unless something bugs out.